cleverpush/src/main/java/com/cleverpush/NotificationOpenedProcessor.java

```diff
@@ -32,18 +32,18 @@ public static void processIntent(Context context, Intent intent) {
         result.setNotification(notification);
         result.setSubscription(subscription);
 
-        CleverPush cleverPush = CleverPush.getInstance(context);
-
-        cleverPush.trackNotificationClicked(notificationId, subscriptionId);
-        cleverPush.fireNotificationOpenedListener(result);
-
         // open launcher activity
         Intent launchIntent = context.getPackageManager().getLaunchIntentForPackage(context.getPackageName());
         if (launchIntent != null) {
             launchIntent.setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT/* | Intent.FLAG_ACTIVITY_NEW_TASK*/);
             context.startActivity(launchIntent);
         }
 
+        CleverPush cleverPush = CleverPush.getInstance(context);
```

Comment:
Is this change necessary? If not please revert.

Reply:
yes if I will open Webview before the launcher activity then it will be not visible to the user launcher activity will cover it

Reply:
Please fix indentation for the whole file

This change is still necessary and will not break users who do not use this feature? Please test.

Reply:
yes this change is necessary as I have explained above and for the scenario in which users not using this feature will not break anything